vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -45,9 +46,10 @@ enum class LogLevel : int32_t {
 
 using OnErrorCallback = Mso::Functor<void(const Mso::ErrorCode &)>;
 using OnLoggingCallback = Mso::Functor<void(LogLevel logLevel, const char *message)>;
-using OnReactInstanceCreatedCallback = Mso::Functor<void(IReactInstance &)>;
-using OnReactInstanceLoadedCallback = Mso::Functor<void(IReactInstance &, const Mso::ErrorCode &)>;
-using OnReactInstanceDestroyedCallback = Mso::Functor<void(IReactInstance &)>;
+using OnReactInstanceCreatedCallback = Mso::Functor<void(Mso::CntPtr<IReactContext> &&, IReactInstance &)>;
```

Comment:
>, IReactInstance & [](start = 86, length = 18)

Should we completely remove it in favor of IReactContext?

Reply:
At some point.  I think its being used in Office right now, so I wanted to keep the old interface until I was sure we didn't need it there.